File: 백준/Bronze/27959. 초코바/초코바.js
const readline = require("readline").createInterface({
    input: process.stdin,
    output: process.stdout
});
readline.question('', input => {
    const[coins, mx] = input.split(' ').map(Number);
    
    if (coins * 100 >= mx) {
        console.log("Yes");
    }
    else{
        console.log("No");
        
    readline.close();
    }
})

File: 백준/Bronze/27959. 초코바/초코바.cs
using System;
class HelloWorld {
  static void Main() {
    
    string[] inputs = Console.ReadLine().Split(' ');
    int coins = int.Parse(inputs[0]);
    int mx = int.Parse(inputs[1]);
    
    if (coins * 100 >= mx) {
        Console.WriteLine("Yes");
    }
    else {
        Console.WriteLine("No");
    }
    
  }
}